### PR TITLE
Update vpn-auto-trigger-profile.md

### PR DESCRIPTION
**Pull request by msarcletti**
Add documentation for the limitation that rasphone.pbk cannot reside in a redirected folder.

### CODE REVIEW

File: windows/security/identity-protection/vpn/vpn-auto-trigger-profile.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ In Windows 10, a number of features were added to auto-trigger VPN so users won
 - Name-based trigger
 - Always On
 
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/vpn/vpn-auto-trigger-profile.md:28]:
```suggestion
> [!NOTE]
```

File: windows/security/identity-protection/vpn/vpn-auto-trigger-profile.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ In Windows 10, a number of features were added to auto-trigger VPN so users won
...
+>Auto-triggered VPN connections will not work if Folder Redirection for %appdata% (C:\Users\username\AppData\Roaming) is enabled. Either Folder Redirection must be disabled for %appdata% or the auto-triggered VPN profile must be deployed in system context, that changes the path the rasphone.pbk file is stored.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/vpn/vpn-auto-trigger-profile.md:29]:
```suggestion
> Auto-triggered VPN connections will not work if Folder Redirection for %appdata% (C:\Users\username\AppData\Roaming) is enabled. Either Folder Redirection must be disabled for %appdata% or the auto-triggered VPN profile must be deployed in system context, which changes the path that the rasphone.pbk file is stored.
```

File: windows/security/identity-protection/vpn/vpn-auto-trigger-profile.md [hunk 3]
```diff
@@ -25,6 +25,10 @@ In Windows 10, a number of features were added to auto-trigger VPN so users won
 - Name-based trigger
 - Always On
 
+> [!NOTE]
+> Auto-triggered VPN connections will not work if Folder Redirection for AppData is enabled. Either Folder Redirection for AppData must be disabled or the auto-triggered VPN profile must be deployed in system context, which changes the path that the rasphone.pbk file is stored.
```
Comment (illfated) [inline windows/security/identity-protection/vpn/vpn-auto-trigger-profile.md:29]: "path to where something is stored"
```suggestion
> Auto-triggered VPN connections will not work if Folder Redirection for AppData is enabled. Either Folder Redirection for AppData must be disabled or the auto-triggered VPN profile must be deployed in system context, which changes the path to where the rasphone.pbk file is stored.
```
Natural daily speech adaption - the phrasing "_the path that something is stored_" is highly unusual.

**Review by illfated — APPROVED** [review]
Thank you for applying the suggestions. 👍🏼

Looks good to me™. 😉

**Pull request merged**